lib/windows/winlean.nim

```diff
@@ -481,6 +481,10 @@ type
     sin6_flowinfo*: int32 # unsigned
     sin6_addr*: In6_addr
 
+  Sockaddr_storage* {.importc: "SOCKADDR_STORAGE",
```

Comment:
This should gain an  'buffer: array[X, byte]' so that the backends which cannot importc types give it a size that doesn't cause stack corruptions.

Reply:
Added field definitions according to https://msdn.microsoft.com/en-us/library/windows/hardware/ff570825(v=vs.85).aspx